charts/seed-bootstrap/dashboards/fluent-bit-dashboard.json

```diff
@@ -1146,10 +1248,42 @@
   "style": "dark",
   "tags": [],
   "templating": {
-    "list": []
+    "list": [
+      {
+        "allValue": "fluent-bit.+",
+        "current": {
+          "selected": true,
+          "tags": [],
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
```

Comment:
These seems to be selecting any `fluent-bit-*` pods found the prometheus. Can it be limited to only that were available in the currently selected time range?

Currently, for the last 3 hours I have only 3 fluent bit pods, however it let me to choose between 20+ pods.

Reply:
Yes... I also tried to do that but did not find good metrics I can use. Loki does watch fluent-bit pod, so I can not use Loki label to get the current pods.
The only way will be to use Prometheus, but did not find a good option there